frontend/src/components/EditRecipe/EditRecipe.js

```diff
@@ -0,0 +1,214 @@
+                <div className='row delete-input'><i class="fas fa-times"></i></div>
+                <div className='row delete-input'><i class="fas fa-times"></i></div>
+                <div className='row delete-input'><i class="fas fa-times"></i></div>
+                <div className='row delete-input'><i class="fas fa-times"></i></div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div className='row add-button'>
+          <div className='col-2 offset-7'>
+            <button type='button' className='btn btn-primary btn-lg'><i class="fas fa-plus"></i>ADD</button>
+          </div>
+        </div>
+        <div className='row directions-title'>
+          <div className='col-8 offset-2'>
+            <div>Directions</div>
+          </div>
+        </div>   
+        <div className='row'>
+          <div className='col-8 offset-2'>
+            <div className='row form-directions'>
+              <div className='col-7 offset-2'>
+                <div className='directions form-group'>
+                  <div className='direction-input'><input className='form-control' type='text' placeholder=''></input></div>
+                  <div className='direction-input'><input className='form-control' type='text' placeholder=''></input></div>
+                  <div className='direction-input'><input className='form-control' type='text' placeholder=''></input></div>
+                  <div className='direction-input'><input className='form-control' type='text' placeholder=''></input></div>
+                  <div className='direction-input'><input className='form-control' type='text' placeholder=''></input></div> 
+                </div>
+              </div>
+              <div className='col-1'>
+                <div className='row delete-input'><i class="fas fa-times"></i></div>
+                <div className='row delete-input'><i class="fas fa-times"></i></div>
+                <div className='row delete-input'><i class="fas fa-times"></i></div>
+                <div className='row delete-input'><i class="fas fa-times"></i></div>
+                <div className='row delete-input'><i class="fas fa-times"></i></div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div className='row add-button'>
+          <div className='col-2 offset-7'>
+            <button type='button' class='btn btn-primary btn-lg'><i class="fas fa-plus"></i>ADD</button>
+          </div>
+        </div>
+        <div className='row photo-title'>
+          <div className='col-8 offset-2'>
+            <div>Photos</div>
+          </div>
+        </div>
+        <div className='row form-photo-upload'>
+          <div className='col-6 offset-3'>
+            <ImageUploader 
+              withIcon={true}
+              buttonText='choose images'
+              onChange={this.onDrop}
+              imgExtension={[ '.jpg', '.gif', '.png' ]}
+              maxFileSize={5242880}
```

Comment:
So I think the goal here is less to have 5 different places to upload, and more to have one upload box and display the pics that have already been uploaded below it. The colors, buttons, and dotted line need a few style changes.